src/AspNet.Security.OAuth.Introspection/OAuthIntrospectionHandler.cs

```diff
@@ -31,12 +31,12 @@ public class OAuthIntrospectionHandler : AuthenticationHandler<OAuthIntrospectio
 
             // Ensure that the authorization header contains the mandatory "Bearer" scheme.
             // See https://tools.ietf.org/html/rfc6750#section-2.1
-            if (!header.StartsWith("Bearer ", StringComparison.OrdinalIgnoreCase)) {
+            if (!header.StartsWith($"{OAuthIntrospectionDefaults.AuthenticationScheme} ", StringComparison.OrdinalIgnoreCase)) {
```

Comment:
True that. You beat me with the spec :+1: Changing the scheme would really make it non-standard.
I guess I just don't like string lying around the code.. :smile:
Soo... I guess we close this PR then.